features/step_definitions/storage_class.rb

```diff
@@ -293,4 +293,15 @@
     logger.error(@result[:response])
     raise "failed to clone StorageClass from: #{src_sc}"
   end  
-end
\ No newline at end of file
+end
+
+Given(/^default storage class exists$/) do
+  ensure_admin_tagged
+  _sc = BushSlicer::StorageClass.get_matching(user: user) { |sc, sc_hash| sc.default? }.first
```

Comment:
@Phaow In general, you should use a single leading underscore only when you need to indicate that a variable, class, method, function, or module is intended for internal use within the containing module, class, or package. Again, this is just a well-established naming convention.